src/middleware/memcached/plugin.js

```diff
@@ -0,0 +1,25 @@
+import Memcached from "memcached";
+
+require("dotenv").config();
+
+export default class MemoryCache {
+  constructor() {
+    const host = process.env.CACHE_HOST || "localhost";
+
+    this.cache = new Memcached(`${host}:11211`);
+  }
+
+  remove(key) {
```

Comment:
**Annotation**: If the existing key is not found, Memcached still resolves the delete method.